state/indexer/block/kv/util.go

```diff
@@ -196,7 +197,7 @@ func dedupMatchEvents(conditions []query.Condition) ([]query.Condition, bool) {
 	for i, c := range conditions {
 		if c.CompositeKey == types.MatchEventKey {
 			// Match events should be added only via RPC as the very first query condition
-			if i == 0 && c.Op == query.OpEqual && c.Operand.(int64) == 1 {
+			if i == 0 && c.Op == query.OpEqual && c.Operand.(*big.Int).Int64() == 1 {
```

Comment:
Same here, this does not appear in the `v0.37.x` version of the patch, do we know the reason?

Reply:
0.37.x does not have `match.events`, hence no `dedupMatchEvents` function.